ffi/dotnet/Devolutions.IronRdp.ConnectExample/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Devolutions.IronRdp;
+namespace Devolutions.IronRdp.ConnectExample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                var serverName = "IT-HELP-DC.ad.it-help.ninja";
+                var username = "Administrator";
+                var password = "DevoLabs123!";
+                var domain = "ad.it-help.ninja";
+
+                Connect(serverName, username, password, domain);
+            }
+            catch (IronRdpException e)
+            {
+                var err = e.Inner.ToDisplay();
+                Console.WriteLine(err);
+            }
+        }
+
+        static void Connect(String servername, String username, String password, String domain)
+        {
+            SocketAddr serverAddr = SocketAddr.LookUp(servername, 3389);
+
+            StdTcpStream stream = StdTcpStream.Connect(serverAddr);
+
+            BlockingTcpFrame tcpFrame = BlockingTcpFrame.FromTcpStream(stream);
+
+            ConfigBuilder configBuilder = ConfigBuilder.New();
+
+            // Password is wrong
```

Comment:
question: Why this comment? I don’t understand the purpose.

Reply:
updated